.github/workflows/render-rmarkdown.yaml

```diff
@@ -0,0 +1,78 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main]
+  pull_request:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+name: render-rmarkdown.yaml
+
+permissions: read-all
+
+jobs:
+  render-rmarkdown:
+    runs-on: ubuntu-latest
+    # Only restrict concurrency for non-PR jobs
+    concurrency:
+      group: rmarkdown-${{ github.event_name != 'pull_request' || github.run_id }}
+    env:
+      BRANCH_NAME: ${{ github.head_ref || github.ref_name }} 
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+      
+    steps:
+      - name: Showing GH context refs
+        shell: bash
+        run: |
+          echo github.head_ref: ${{ github.head_ref }}
+          echo github.ref: ${{ github.ref }}
+          echo github.ref_name: ${{ github.ref_name }}
+          echo github.base.ref: ${{ github.base_ref }}
+          
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+          fetch-depth: 0
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 'latest'
+
+      - name: Setup Pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+
+      - name: Setup R Dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::rmarkdown
```

Comment:
As I understand it, this step takes care of installing the local `blazr` package.
See: https://github.com/r-lib/actions/tree/v2-branch/setup-r-dependencies

Reply:
This is probably handled 
- `uses: r-lib/actions/setup-r-dependencies@v2
  with:
    extra-packages: any::rmarkdown, local::.`
    

Note to self: See [here](https://github.com/r-lib/actions/blob/v2-branch/examples%2FREADME.md)

Reply:
Resolved. See commit message below.